analyzers/tests/SonarAnalyzer.UnitTest/Rules/SqlKeywordsDelimitedBySpaceTest.cs

```diff
@@ -73,6 +73,13 @@ public class SqlKeywordsDelimitedBySpaceTest
                 .WithOptions(ParseOptionsHelper.FromCSharp11)
                 .Verify();
 
+        [TestMethod]
+        public void SqlKeywordsDelimitedBySpace_CSharp12() =>
+            builder.AddPaths("SqlKeywordsDelimitedBySpace.CSharp12.cs")
+                .WithOptions(ParseOptionsHelper.FromCSharp12)
+                .WithConcurrentAnalysis(false)
```

Comment:
Concurrent analysis requires the introduction of an enclosing namespace, altering the code being compiled.
That namespace conflicts with the file-scoped namespace I had to introduce, to avoid False Negatives, since lack of namespace (or better, the use of the default one), confuses the implementation of the rule.

I could have defined a non-file-scoped namespace in `SqlKeywordsDelimitedBySpace.CSharp12.cs`, but that would have added an unnecessary level of nesting. Because concurrent analysis doesn't seem to be relevant for the test cases here, I have instead opted for disabling it.